playground/update.sh

```diff
@@ -1,24 +1,39 @@
-#! /bin/sh
+#!/bin/bash
```

Comment:
I don't actually see anything in this script that would actually require bash over sh. Keeping to sh would make it easier for some BSDs to run this script, as they don't have Bash installed by default.